src/css/styles.css

```diff
@@ -1,24 +1,87 @@
-.icon {
-  width: 60px;
-  height: 60px;
+.color-box div {
+  display: inline-block;
+  margin: 10px;
+  width: 120px;
+  height: 100px;
+  border: dashed 1px black;
+  text-align: center;
+  line-height: 100px;
 }
 
-.icon-wrapper {
-  font-size: 12px;
-  text-shadow: 2px 2px #444444;
-  overflow: hidden;
-  width: 80px;
-  height: 80px;
-  background: linear-gradient(to bottom, white 0%,#744662 75%,#444444 76%,#444444 100%);
-  background-color: #dddddd !important;
+ul.native-list {
+  padding: 0 30px;
 }
 
-.dropdown-menu {
-  width: 266px;
-  background: linear-gradient(#744662, #dddddd);
-  background-color: #ababab !important;
```

Comment:
you are deleting previous styles...please revert these changes. Generally, it is ok, to use additional styling in your components, but since we installed bootstrap, I suggest using the bootstrap default styles atm. We need to keep styling consistent.